### DISCUSSION TITLE
Asking Collection location everytime

### DISCUSSION THREAD

**Discussion by akshat-khosya** [usebruno/bruno#1232] [Ideas]
Whenever we want to create a collection we ask collection name along with folder path,  I think we can avoid that.
We can store all collection in place like Windows: AppData folder , Linux : .local, .config and mac: library.
Once user want to share with anyone either we can have an exported JSON or  Zip of that folder. 
What you guys think of it.
@helloanoop @Its-treason

**Comment by Its-treason** [comment]
I'm not really sure about the saving collection in `~/.config` thing, but I do understand the point about having to enter the location every time being kinda annoying. I think having path suggestions that are based on currently open collections, would be kinda nice.
